lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8866,15 +8869,21 @@ void ClangImporter::Implementation::importSpanAttributes(FuncDecl *MappedDecl) {
                              ->getDesugaredType()
                              ->getString()));
     }
+    bool lifetimeDependenceOn = MappedDecl->getASTContext().LangOpts.hasFeature(
```

Comment:
There's no test for this right?

Reply:
Good catch. There is a test `test/Macros/SwiftifyImport/CxxSpan/NoEscapeSpan.swift` where this feature is not turned on but I don't have any lifetime dependency annotations there. I can add some.

Reply:
OH wait, you mean testing the transformation end to end. My bad. I totally forgot that. Will add one.